src/engine.rs

```diff
@@ -245,6 +249,36 @@ impl Engine {
         );
     }
 
+    /// There is one Aurora block per NEAR block height (note: when heights in NEAR are skipped
+    /// they are interpreted as empty blocks on Aurora). The blockhash is derived from the height
+    /// according to
+    /// ```text
+    /// block_hash = sha256(concat(
+    ///     BLOCK_HASH_PREFIX,
+    ///     block_height as u64,
+    ///     chain_id,
+    ///     engine_account_id,
+    /// ))
+    /// ```
+    pub fn compute_block_hash(chain_id: [u8; 32], block_height: u64, account_id: &[u8]) -> H256 {
+        let mut data = Vec::with_capacity(1 + 8 + 32 + account_id.len());
```

Comment:
Avoiding magic numbers. I suggest to replace them with predefined constants: 

-`BLOCK_HASH_PREFIX_SIZE` -> `1`
-`BLOCK_HEIGHT_SIZE` -> `8`
-`CHAIN_ID_SIZE` -> `32`

Reply:
Good call. Done in https://github.com/aurora-is-near/aurora-engine/pull/213/commits/3334f18a944cf97048cde9f3b38a67db7706fc63